source/import-css/index.html.md.erb

```diff
@@ -40,20 +40,68 @@ To import all the Sass rules from GOV.UK Frontend, add the following to your Sas
 If you want to improve how quickly your service's pages load in browsers, you can import only the Sass rules you need.
 
 1. Import `node_modules/govuk-frontend/dist/govuk/base` in your Sass file.
-2. Import the parts of the CSS you need.
+2. Import only the Sass that you need, based on what components and other classes are being used in your service.
 
-For example, add the following to your Sass file to import the CSS you need for a basic GOV.UK page.
+The biggest optimisations come from excluding any components or overrides that you aren't using.
+
+You must import the layers in the order listed in the example below.
 
 ```scss
+// 'Base' includes everything from settings, tools and helpers. Nothing
+// in the base outputs any actual CSS.
+
 @import "node_modules/govuk-frontend/dist/govuk/base";
 
+// Basic content styles for typography, links etc. Approximately 10% of
+// the CSS output if you include everything.
+
 @import "node_modules/govuk-frontend/dist/govuk/core/index";
+
+// Objects include things like the page template, grid, form groups.
+// Approximately 5% of the CSS output if you include everything.
+
 @import "node_modules/govuk-frontend/dist/govuk/objects/index";
-@import "node_modules/govuk-frontend/dist/govuk/components/footer/index";
-@import "node_modules/govuk-frontend/dist/govuk/components/header/index";
-@import "node_modules/govuk-frontend/dist/govuk/components/skip-link/index";
+
+// The components themselves - try to only include the components you
+// are using in your project. Approximately 70% of the CSS output if
+// you include everything.
+
+<% components.each do |component| %>
+@import "node_modules/govuk-frontend/dist/govuk/components/<%= component %>/index";
+<% end %>
+
+/*
+// Alternatively, you can import all components:
+@import "node_modules/govuk-frontend/dist/govuk/components/index";
+*/
+
+// Utilities, for example govuk-clearfix or govuk-visually-hidden.
+// Approximately 1% of the CSS output if you include everything.
+
 @import "node_modules/govuk-frontend/dist/govuk/utilities/index";
+
+// Overrides, used to tweak things like the amount of spacing on an
```

Comment:
```suggestion
// Overrides, used to amend things like the amount of spacing on an
```

Reply:
Happy with 'adjust' instead? 'amend' doesn't feel natural in this context.